src/NServiceBus.Core.Tests/ApprovalFiles/APIApprovals.ApproveNServiceBus.netframework.approved.txt

```diff
@@ -65,8 +65,12 @@ namespace NServiceBus
     {
         public static NServiceBus.DataBus.DataBusExtensions<NServiceBus.FileShareDataBus> BasePath(this NServiceBus.DataBus.DataBusExtensions<NServiceBus.FileShareDataBus> config, string basePath) { }
     }
+    [System.ObsoleteAttribute("Will be treated as an error from version 8.0.0. Will be removed in version 9.0.0." +
```

Comment:
ummm what's with the `..." + ""` thingy in all the usages down below? Shouldn't that just be:

`[System.ObsoleteAttribute("Will be treated as an error from version 8.0.0. Will be removed in version 9.0.0.", false)]`

Reply:
Seems to be a quirk of `PublicApiGenerator`